bin/cli.js

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env node
+'use strict';
+
+const program = require('commander');
+const common = require('metarhia-common');
+const karma = require('karma');
+const yaml = require('yaml').default;
+const path = require('path');
+const fs = require('fs');
+
+const splitOpts = v => v.split(',');
+
+const cliOptions = [
+  [ '--exclude <patterns>',      'Exclude tests patterns', splitOpts, [] ],
+  [ '--browsers <values>',       'Browsers to run', splitOpts, [] ],
+  [ '--reporter <value>',        'Reporter name'],
+  [ '--log-level <value>',       'Log level'],
+  [ '-p, --browser-port <n>',    'Browser port' ],
+  [ '-c, --config <value>',      'Config file' ],
+];
+
+const karmaLogLevels = {
+  'disable': [ 'LOG_DISABLE', 'none' ],
+  'error':   [ 'LOG_ERROR', 'errors-only' ],
+  'warn':    [ 'LOG_WARN', 'minimal' ],
+  'info':    [ 'LOG_INFO', 'normal' ],
+  'debug':   [ 'LOG_DEBUG', 'verbose' ],
+};
+
+const browserLaunchers = {
+  'Chrome':         'karma-chrome-launcher',
+  'ChromeHeadless': 'karma-chrome-launcher',
+  'Firefox':        'karma-firefox-launcher',
+  'IE':             'karma-ie-launcher',
+  'Opera':          'karma-opera-launcher',
+  'Safari':         'karma-safari-launcher',
+};
+
+const logLevels = ['quiet', 'disable', 'error', 'warn', 'info', 'debug'];
+
+const isLogAtLeast = (level1, level2) => {
+  if (logLevels.indexOf(level1) === -1) level1 = 'disable';
```

Comment:
```suggestion
  if (!logLevels.includes(level1)) level1 = 'disable';
```
But don't change arguments

Reply:
But I'd prefer not to change arguments

Reply:
`disable` is not the lowest log level. That is `quiet`, but it shouldn't be the default. Also, I don't think that the default value should be bonded to its index as we might add more log levels in the future and this will require reverting this back to its current state.